diversity/mentoring/index.md

```diff
@@ -4,7 +4,72 @@ layout: page
+  
+## How do I apply?
+
+If you are interested in the program and eligible for it, please apply using this form: https://forms.gle/aLuFtxg6Fe8KatEL8.
+ 
+## FAQ
+
+* Q. How many papers are supported in the pilot program in 2022? Is there a chance that my application for mentorship might not be accommodated? 
+<br> <br> *A. Since the program is in a pilot stage in 2022, the number of papers we support is limited by our current organisational capacity and availability of mentors. It may be possible that not all applications are accommodated if we receive a large number of requests. However, authors will know regarding acceptance of their applications before submitting papers to the program.* 
+<br>
+
+* Q. Who will be my mentor if I apply to the program? 
+<br> <br> *A. Invited senior members from the MIR community have been kind enough to volunteer for the program. The mentors and mentees will be matched based on the topic of papers as chosen during submission and introduced to each other.*
+<br>
+
+* Q. If I incorporate all feedback from the mentor into my paper, will my paper be accepted for publication in the ISMIR conference? Is there a different review process for the mentored papers? 
+<br> <br> *A. All mentored papers will go through the same review process as all other papers submitted to an ISMIR conference. There is no assurance that a mentored paper will be accepted to the conference.*
 <br>
-The New-to-ISMIR paper mentoring program program is designed for members new to MIR topics (early stage researchers in MIR or researchers from allied fields who wish to submit their work to an ISMIR conference) to share their advanced-stage work-in-progress ISMIR paper drafts with senior members of the ISMIR community (mentors) to obtain focused reviews and constructive feedback. The program is expected to supplement the already available generic submission guidelines and resources with paper specific feedback. The program is being run as a pilot in 2022, closely aligned with the ISMIR 2022 paper submissions deadlines.  
 
-More information coming soon.
+For any further questions, please write to us at <a href="mailto:ismir2022-diversity@ismir.net">ismir2022-diversity@ismir.net</a>

```

Comment:
Could you please format this part to markdown instead of HTML

Reply:
Done. Pls have a look.